cmd/swarmctl/network/common.go

```diff
@@ -32,6 +32,12 @@ func GetNetwork(ctx context.Context, c api.ControlClient, input string) (*api.Ne
 		}
 
 		if l := len(rl.Networks); l > 1 {
+			for _, n := range rl.Networks {
+				if n.Spec.Annotations.Name == input {
+					// Found a full name match
```

Comment:
I just found that it is indeed possible that there are multiple nodes with the same name but in different status.
49kmbx2yxf63zn801joyunzm4  node-3  ACCEPTED    READY   ACTIVE        
5a2m77pyawva7b5jjmzmmadjn  node-3  ACCEPTED    DOWN    ACTIVE

Do you know why server side would allow the creation of nodes with the same name? Is it a bug?